gallery.html

```diff
@@ -2,10 +2,67 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="author" content="Juan Nunez" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Gallery</title>
+    <!--Font Links Section-->
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <!--CSS Links Section-->
+    <link rel="stylesheet" href="index.css">
+
+    <title>Juan's Gallery</title>
 </head>
 <body>
+    <header>
+        <nav class="topnav">
+            <ul class="navul">             
+                <li class="navli"><a href="index.html">Home</a></li>|
+                <li class="navli"><a href="gallery.html">Gallery</a><li>|
+                <li class="navli"><a href="contact.html">Contact me</a></li>
+            </ul>
+        </nav>
+    </header>
+    
+    <main>
+        <center><h1>My favorite collection of cars</h1></center>
+
+        <div class="gallery">
+            <center>
+            <div class="column"><img src="images/A7.jpg" alt="Picture of an Audi A7" width="200"></div>
+            <div class="column"><img src="images/c10.jpg" alt="Picture of a silverado c10" width="200"></div>
+            <div class="column"><img src="images/impala.jpg" alt="Picture of an old school impala " width="200" ></div>
+            </center>
+        </div>
+        <br><br>
```

Comment:
What led to adding these 'br' tags?  Deleting these tags doesn't seem to change the way the page is rendered.  If that's the case, I would recommend removing them -- avoid any code which doesn't have a clear desired behavior.